src/commands/crashlytics-mappingfile-upload.ts

```diff
@@ -0,0 +1,73 @@
+import { Command } from "../command";
+import { FirebaseError } from "../error";
+import * as utils from "../utils";
+
+import { fetchBuildtoolsJar, runBuildtoolsCommand } from "../crashlytics/buildToolsJarHelper";
+import { Options } from "../options";
+
+interface CommandOptions extends Options {
+  app?: string;
+  mappingFile?: string;
+  resourceFile?: string;
+}
+
+interface JarOptions {
+  app: string;
+  mappingFilePath: string;
+  resourceFilePath: string;
+}
+
+export const command = new Command("crashlytics:mappingfile:upload <mappingFile>")
+  .description("upload a ProGuard/R8-compatible mapping file to deobfuscate stack traces")
+  .option("--app <appID>", "The app id of your Firebase app")
+  .option(
+    "--resource-file <resourceFile>",
+    "Path to the Android resource XML file that includes the mapping file id."
+  )
+  .option("--debug", "print debug output and logging from the underlying uploader tool")
```

Comment:
This is a global option - it shouldn't need to be redefined and should be able to be safely removed